### DISCUSSION TITLE
Record Multiple Frames on Camera

### DISCUSSION THREAD

**Discussion by SoppierBrute9** [bonsai-rx/bonsai#1530] [Q&A]
I have followed tutorial on the bonsai website to video capture, but it is only capturing one frame when I start it. How do I make a workflow such that when I start the workflow it begins recording and stops recording when I stop the workflow? Also how can this be done for three cameras simultaneously? I guess the better way is what is recorded is compressed into one second in the .avi file.

**Comment by RoboDoig** [comment]
What type of camera are you using? Recording when the workflow starts and stopping when the workflow ends is usually the default behaviour. Can you send the workflow file you are using?
